client/client_test.go

```diff
@@ -155,3 +155,41 @@ func (cs *clientSuite) TestClientCapabilities(c *check.C) {
 		},
 	})
 }
+
+func (cs *clientSuite) TestClientAddCapability(c *check.C) {
+	cs.rsp = `{
+		"type": "sync",
+		"result": {
+		}
+	}`
+	cap := &client.Capability{
+		Name:  "n",
+		Label: "l",
+		Type:  "t",
+		Attrs: map[string]string{"k": "v"},
+	}
+	err := cs.cli.AddCapability(cap)
+	c.Check(err, check.IsNil)
+}
+
+func (cs *clientSuite) TestClientRemoveCapabilityOk(c *check.C) {
+	cs.rsp = `{
+		"type": "sync",
+		"result": { }
+	}`
+	err := cs.cli.RemoveCapability("n")
```

Comment:
As mentioned in the other branch, would this test pass if the whole body is replaced by `return nil`?